driver.json

```diff
@@ -27,12 +27,14 @@
 				"label": {
 					"en": "Setup process",
 					"de": "Setup Fortschritt",
-					"fr": "Progrès de la configuration"
+					"fr": "Progression de la configuration"
 				},
 				"field": {
 					"label": {
 						"value": {
-							"en": "The integration will discover your Android TV on your network. Only devices running the [Android TV Remote Service](https://play.google.com/store/apps/details?id=com.google.android.tv.remote.service) are supported.\nDuring the process, you might need to enter a PIN that is shown on your Android TV. Please make sure that your Android TV is powered on and that no old pairing request is shown.\nIf pairing continuously fails, reboot your Android TV device and try again."
+							"en": "The integration will discover your Android TV on your network. Only devices running the [Android TV Remote Service](https://play.google.com/store/apps/details?id=com.google.android.tv.remote.service) are supported.\nDuring the process, you might need to enter a PIN that is shown on your Android TV. Please make sure that your Android TV is powered on and that no old pairing request is shown.\nIf pairing continuously fails, reboot your Android TV device and try again.",
+							"de": "Die Integration wird dein Apple TV in deinem Netzwerk erkennen.\nDurch die Integration wird Ihr Android TV in Ihrem Netzwerk erkannt. Es werden nur Geräte mit [Android TV Remote Service](https://play.google.com/store/apps/details?id=com.google.android.tv.remote.service) unterstützt\nWährend des Vorgangs müssen Sie möglicherweise eine PIN eingeben, die auf Ihrem Android TV angezeigt wird. Bitte stellen Sie sicher, dass Ihr Android TV eingeschaltet ist und keine alte Kopplungsanfrage angezeigt wird.\nWenn die Kopplung ständig fehlschlägt, starten Sie Ihr Android TV-Gerät neu und versuchen Sie es erneut.",
```

Comment:
German translation is in PR #32, please remove to avoid conflicts. (And it contains mixed formal and informal pronouns ;-)

Reply:
Removed. I kept the French updates